src/order/initial_seg.lean

```diff
@@ -312,6 +313,22 @@ def of_element {α : Type*} (r : α → α → Prop) (a : α) : subrel r {b | r
 @[simp] theorem of_element_top {α : Type*} (r : α → α → Prop) (a : α) :
   (of_element r a).top = a := rfl
 
+/-- Any principal segment of a well order is order-isomorphic to a `subrel` -/
```

Comment:
I don't understand this docstring; which side of the isomorphism is the "principal segment"?